site/zenodo_rdm/redirector.py

```diff
@@ -0,0 +1,221 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Redirector functions and rules."""
+
+from flask import request, url_for
+
+ZENODO_TYPE_SUBTYPE_LEGACY = {
```

Comment:
This could be a config and then use `current_app.get()` but since it was already _not_ being used... I didnt